tests/network-tests/tsconfig.json

```diff
@@ -3,12 +3,13 @@
     "target": "esnext",
     "module": "commonjs",
     "outDir": "dist",
-    "rootDir": "src",
+    "rootDirs": ["src", "../../query-node/mappings/src/content/utils"],
```

Comment:
Why is this needed? We are surely not using `utils.ts` as root, or do we?

Reply:
Was getting the following error while building `network-tests`:
```
File '/workspace/joystream/query-node/mappings/src/content/utils.ts' is not under 'rootDir' '/workspace/joystream/tests/network-tests/src'. 'rootDir' is expected to contain all source files.ts(6059)
```
So made the change.

_Update:_
Just found out that the above error disappears if change 
```javascript
"rootDir": "src"
```
to 
```javascript
"rootDirs": ["src"]
```
:shrug: